src/ccb_core/database_config.cc

```diff
@@ -83,8 +83,8 @@ database_config::database_config(config::endpoint const& cfg) {
   if (it != end)
     _type = it->second;
   else
-    throw exceptions::config()
-          << "no 'db_type' defined for endpoint '" << cfg.name << "'";
+    throw exceptions::config("no 'db_type' defined for endpoint '{}'",
+                             cfg.name);
```

Comment:
It is strange that the com::centreon namespace is not marked to be used and you can call exceptions::config from it.
Could you, use the com::centreon::exceptions namespace and then you can remove "exceptions" from this call.